src/Core/Localization/CLDR/LocaleData.php

```diff
@@ -106,6 +106,13 @@ class LocaleData
      */
     protected $currencies;
 
+    /**
+     * All territories, by ISO code.
+     *
+     * @var TerritoryData[]
+     */
+    protected $territories = [];
```

Comment:
```suggestion
     * @var TerritoryData[]|null
     */
    protected $territories;
```

Reply:
To be consistent with other fields and https://github.com/PrestaShop/PrestaShop/pull/21680

Reply:
Fixed.